CONTRIBUTING.md

```diff
@@ -50,8 +50,7 @@ Use [black](https://github.com/ambv/black) to make sure the code follows the sty
 
 This custom integration is based on [integration_blueprint template](https://github.com/custom-components/integration_blueprint).
 
-It comes with development environment in a container, easy to launch
-if you use Visual Studio Code. With this container you will have a stand alone
+It comes with development environment in a container that is easy to launch if you use Visual Studio Code. With this container you will have a stand alone.
```

Comment:
Last sentence is a fragment. Remove period.

Reply:
done sir